sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/alibaba.yaml

```diff
@@ -64,30 +69,29 @@ provider:
         - text: |
             String aliyunAccessKeySecret = "100000000000000000000000000000";
           containsSecret: false
+
+    - rspecKey: S6336
+      id: alibaba-access-key-secrets-2
       detection:
+        pre:
+          include:
+            content:
+              - aliyun
+              - alibaba
+              - alicloud
         matching:
-          pattern: "(?i)(?<![A-Z])ali(?:yun|baba|cloud).{0,50}['\"`]([0-9a-z]{30})['\"`]"
+          pattern: "(?i)(?:SECRET_?(?:ACCESS)?_?KEY|(?:ACCESS)?_?KEY_?SECRET)\\b[^0-9a-z]{0,10}([0-9a-z]{30})(?![a-z0-9\\/+=$\\-_])"
```

Comment:
What is the why for this change ? you removed pretty important prefixes here

Reply:
No, this is only a weird diff display. The structure of the file changed a bit. The actual changes to the regex are minor.

Reply:
Ah ok I see